x-pack/legacy/plugins/siem/index.ts

```diff
@@ -144,6 +146,19 @@ export const siem = (kibana: any) => {
           category: ['siem'],
           requiresPageReload: true,
         },
+        [IP_REPUTATION_LINKS_SETTING]: {
+          name: i18n.translate('xpack.siem.uiSettings.ipReputationLinks', {
+            defaultMessage: 'IP Reputation Links',
+          }),
+          value: IP_REPUTATION_LINKS_SETTING_DEFAULT,
```

Comment:
Any idea if we can do anything about the `IP Reputation` default value overflowing into the second column @angorayc? When looking around the [source for field type format](https://github.com/elastic/kibana/blob/69fc4437ff6ef227de14dd859a755dd11281973f/src/legacy/core_plugins/kibana/ui_setting_defaults.js#L706) I'm not seeing any additional config that specifies an explicit width, so we may want to open a core kibana issue for this.

Our lengthy string default values look to be the cause (`field type format` has multiple spaces it can break on). I imagine the ideal fix would be to keep the column widths static and just have a scroll bar for the `default value` that is displayed?

#### Field type format name without overflow:
<img src="https://user-images.githubusercontent.com/2946766/74865984-49715380-530f-11ea-98e0-d08dcfe0e97e.png" width=300>




#### IP Reputation Link with overflow:
<img src="https://user-images.githubusercontent.com/2946766/74866007-50986180-530f-11ea-8ac6-57b16039193a.png" width=300>

Reply:
Ahh, good catch! Thank you! I've created an issue and put it in [Kibana's channel](https://elastic.slack.com/archives/C0D8P2XK5/p1582194738095600)
https://github.com/elastic/kibana/issues/58102